core/sql/udrserv/udrserv.cpp

```diff
@@ -230,35 +230,6 @@ void InitializeJavaOptionsFromEnvironment(LmJavaOptions &javaOptions
 {
    char *val;
 
-   if ((val = getenv("CLASSPATH")))
-   {
-      javaOptions.addSystemProperty("java.class.path", val);
-   }
-   char maxHeapOption[64];
-   int maxHeapEnvvarMB = DEFAULT_JVM_MAX_HEAP_SIZE;
```

Comment:
I pushed [another commit](https://github.com/zellerh/incubator-trafodion/commit/8b38be76203dabcb47a03c09a72b14bd305d1564) to remove the 3 lines, but for some reason it does not show up here. If the rework doesn't get committed I'll fix that in a later PR.